src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_state.go

```diff
@@ -10,6 +10,20 @@ import (
 	util "github.com/filecoin-project/specs/util"
 )
 
+func (st *StoragePowerActorState_I) ActivePowerMeetsConsensusMinimum(minerPower block.StoragePower) bool {
+	totPower := st._getActivePower()
```

Comment:
nit: call this `totalActivePower` instead of `totPower`?

Reply:
feel it's obvs from function names all around :b. Will do if I touch up again!